### PR TITLE
Move Azure IPAM out of beta

### PR DESCRIPTION
**Pull request by tgraf**

### CODE REVIEW

File: Documentation/gettingstarted/k8s-install-azure.rst [hunk 1]
```diff
@@ -26,7 +21,47 @@ Setup a Kubernetes cluster on Azure. You can use any method available as long
...
+
+.. note:: **Do NOT specify the '--network-policy' flag** when creating the
+    cluster, as this will cause the Azure CNI plugin to push down unwanted
+    iptables rules.
+
+.. code:: bash
+
+   export RESOURCE_GROUP_NAME=aks-test
+   export CLUSTER_NAME=aks-test
+   export LOCATION=westeurope
```
Comment (twpayne) [inline Documentation/gettingstarted/k8s-install-azure.rst]: Tiny nit: the previous command uses location `westus` but this command uses `westeurope`.

File: Documentation/gettingstarted/cni-chaining-azure-cni.rst [hunk 2]
```diff
@@ -8,34 +8,123 @@
...
+allocation (IPAM). After the initial networking is setup, the Cilium CNI plugin
+is called to attach eBPF programs to the network devices set up by Azure CNI to
+enforce network policies, perform load-balancing, and encryption.
+
+
+Create an AKS + Cilium CNI configuration
+========================================
+
+Create a ``chaining.yaml`` file based on the following template to specify the
+desired CNI chaining configuration:
```
Comment (twpayne) [inline Documentation/gettingstarted/cni-chaining-azure-cni.rst]: Is it worth adding a sentence or two describing what `chaining.yaml` does? For example, the need for the `azure-vnet` and `ciluim` plugins is clear, but why is the `portmap` plugin needed?

**Review by twpayne — APPROVED** [review]
LGTM, two minor comments - feel free to ignore them.

**Review by qmonnet — CHANGES_REQUESTED** [review]
It looks like the following links will break, can you please update them?
```
$ grep k8s-install-aks install | sed 's/:.*\[/: [/'     
install/kubernetes/cilium/README.md.gotmpl: [Microsoft AKS](https://raw.githubusercontent.com/cilium/charts/master/images/azure.svg)](https://cilium.readthedocs.io/en/stable/gettingstarted/k8s-install-aks/) |
install/kubernetes/cilium/README.md: [Microsoft AKS](https://raw.githubusercontent.com/cilium/charts/master/images/azure.svg)](https://cilium.readthedocs.io/en/stable/gettingstarted/k8s-install-aks/) |
```

Looks good to me otherwise.

**Comment by tgraf** [comment]
test-docs

**Review by jibi — APPROVED** [review]

**Review by qmonnet — APPROVED** [review]

**Review by ti-mo — CHANGES_REQUESTED** [review]
Looks good! I left another mention of Azure IPAM being beta in `gettingstarted/kubeproxy-free.rst` at the time, which probably needs to be removed:

> Deploying Cilium with `azure.enabled=true` will disconnect all running Pods in the cluster that were scheduled using the default azure-vnet CNI plugin. We aim to remove this limitation before moving Azure IPAM out of Beta.

(:point_up: Reading this again, it's not 100% accurate anyway, as this only manifests on legacy nodepools configured in bridge mode.)

There's also a `helm install` example using Azure IPAM in `kubeproxy-free.rst` that might need to be replaced with a link to the AKS install guide or an rst include to prevent it from going stale.

**Review by errordeveloper — APPROVED** [review]

**Review by ti-mo — APPROVED** [review]
Looks good!

**Comment by jibi** [comment]
I'm marking this as ready-to-merge without full CI tests as the PR is only touching documentation

**Pull request merged**